packages/console/src/assets/docs/guides/native-flutter/README.mdx

```diff
@@ -1 +1,214 @@
-## Replace this with actual guide
\ No newline at end of file
+import UriInputField from '@/mdx-components-v2/UriInputField';
+import Steps from '@/mdx-components-v2/Steps';
+import Step from '@/mdx-components-v2/Step';
+import Tabs from '@mdx/components/Tabs';
+import TabItem from '@mdx/components/TabItem';
+import InlineNotification from '@/ds-components/InlineNotification';
+
+<Steps>
+<Step title="Install Logto SDK">
+
+Our flutter SDK package is published on [pub.dev](https://pub.dev/packages/logto_dart_sdk). Install the SDK package by running the following command in your project directory:
+
+```bash
+flutter pub get logto_dart_sdk
+```
+
+</Step>
+<Step title="Dependency settings" subtitle="2 steps">
+<details>
+
+<summary>flutter_secure_storage</summary>
+
+<p>
+
+We use [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage) to implement the cross-platform persistent secure token storage.
+
+- Keychain is used for iOS
+- AES encryption is used for Android.
+
+### Config Android version:
+
+In [project]/android/app/build.gradle set minSdkVersion to >= 18.
+
+```gradle
+android {
+    ...
+
+    defaultConfig {
+        ...
+        minSdkVersion 18
+        ...
+    }
+}
+```
+
+<InlineNotification>
+
+By default Android backups data on Google Drive. It can cause exception java.security.InvalidKeyException:Failed to unwrap key. You will need to:
+
+    - disable autobackup,
+    - or exclude sharedprefs FlutterSecureStorage used by the plugin
+
+</InlineNotification>
+
+1. To disable auto backup, go to your app manifest file and set the boolean value android:allowBackup:
```

Comment:
in general, the code or non natural language tech stuff should be wrapped in the backquotes. there are few more places have the same issue in this article

```suggestion
1. To disable auto backup, go to your app manifest file and set the boolean value `android:allowBackup`:
```